src/SSVOpenHexagon/Core/HGUpdate.cpp

```diff
@@ -25,14 +25,28 @@ void HexagonGame::update(ssvu::FT mFT)
 {
     mFT *= Config::getTimescale();
 
-    // TODO: refactor to avoid repetition, and truncate floating point number
-    // TODO: also show best record (here) and last run + best record (in menu)
-    steamManager.set_rich_presence_in_game(
-        levelData->name, status.getTimeSeconds());
+    // TODO: show best record (here) and last run + best record (in menu)
+
+    // Name formatter
+    std::string nameStr = std::string(levelData->name);
+    nameStr[0] = std::toupper(nameStr[0]);
+
+    // Difficulty multipler formatter
+    std::string diffStr = std::to_string(HexagonGame::difficultyMult);
```

Comment:
`HexagonGame::difficultyMult` is redundant, just say `difficultyMult`.

```suggestion
    std::string diffStr = std::to_string(difficultyMult);
```